taxonomy/algolia/serializers.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+"""
+Serializers to convert db data to a dictionary that algolia can understand.
+
+This module depends on serializers provided by django-rest-framework.
+"""
+
+from rest_framework import serializers
+
+from taxonomy.models import Job, JobPostings, JobSkills
+
+
+class JobPostingSerializer(serializers.ModelSerializer):
+    """
+    JobPosting serializer for algolia index.
+
+    This serializer will contain all of the metadata related to the job posting.
+    """
+    class Meta:
+        model = JobPostings
+        fields = (
+            'job_id', 'median_salary', 'median_posting_duration', 'unique_postings', 'unique_companies',
```

Comment:
Why are `median_posting_duration` and `unique_companies` being serialized, I don't think these fields are required on frontend.

Reply:
I think it is better to add this information, in case we need it in the future.